arangograph/scim.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+description: How to enable SCIM provisioning
+title: SCIM Provisioning
+---
+# SCIM Provisioning
+
+{% hint 'info' %}
+This feature is only available in ArangoGraph Enterprise.
+{% endhint %}
+
+ArangoGraph provides support to control and manage members access in
+ArangoGraph organizations with the
+**System for Cross-domain Identity Management** (SCIM) provisioning. 
+This enables you to propagate to ArangoGraph any user access changes by using
+the dedicated API.
+
+{% hint 'info' %}
+To enable the SCIM feature, get in touch with the ArangoGraph team via an
+ArangoGraph Support Ticket.
```

Comment:
Would be good to have link to Support page, here.

Reply:
since we don't have a generic link, rephrased this and pointed to the Request Help form.